pkg/api/submit.proto

```diff
@@ -62,6 +62,17 @@ message JobCancelRequest {
     string queue = 3;
 }
 
+// swagger:model
+message JobSetCancelRequest {
+    string job_set_id = 1;
+    string queue = 2;
+    JobSetFilter filter = 3;
+}
+
+message JobSetFilter {
+    repeated string state = 1;
```

Comment:
Can you make this a enum instead of a raw string?

Reply:
Yeah that was my thought too, will go over the public API change with Chris today to decide what we want
 - This being an enum was a point I mentioned to him already